src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -100,6 +100,30 @@ public function expiresAfter($time)
     }
 
     /**
+     * Validates a cache key according to PSR-6.
+     *
+     * @param string $key The key to validate.
+     *
+     * @return string $key if it is valid.
+     *
+     * @throws InvalidArgumentException When $key is not valid.
+     */
+    public static function validateKey($key)
```

Comment:
I don't understand this comment. We can return a boolean and throw exceptions. Some Symfony methods already do that.

---

Anyway, maybe the problem is with the method name, because it's not a validator but a normalizer (in theory it could change the value of the key to make it valid).